### ISSUE TITLE
Entity supports the turn_on/turn_off methods without setting the proper ClimateEntityFeature

### ISSUE THREAD

**Issue by JeroenTogt** [zxdavb/ramses_cc#200]
HVAC modes are not the actual modes the FAN is capable of
FAN is Itho CVE-S ECO

Warning in the log file during start:

`2024-08-17 18:40:13.470 WARNING (MainThread) [homeassistant.components.climate] Entity climate.37:252531 (<class 'custom_components.ramses_cc.climate.RamsesHvac'>) implements HVACMode(s): auto, off and therefore implicitly supports the turn_on/turn_off methods without setting the proper ClimateEntityFeature. Please create a bug report at https://github.com/zxdavb/ramses_cc/issues`

Usage case is the following:

1. Setting modes of fan 37:252531 with the fysical remote 37:012161,  both fan and remote are a "37"
2. Setting modes with faked remote with same id as fysical remote
3. Automation that switces fan on via a zwave plug and repeats the command as with putting power on the FAN the fan starts always in auto mode.

The way I have implemented works fine but obviously the controls that are made for the FAN entity are not working and it does give the warning in the LOG..

For me just a nuisance in the log, but there are probably some tweeks to fix it or recognize the Itho CVE-S ECO

Relevant part of Schema:

```
orphans_hvac:
  - "37:012161"
  - "37:252531"
```

Relevant part of know device id's

```
"37:012161":
  class: REM
  faked: true
  commands:
    Auto_Mode: " I --- 37:012161 37:252531 --:------ 22F1 003 000304"
    Laag_Stand: " I --- 37:012161 37:252531 --:------ 22F1 003 000204"
    Hoog_Stand: " I --- 37:012161 37:252531 --:------ 22F1 003 000404"
    10Min_Hoog: " I --- 37:012161 37:252531 --:------ 22F3 003 00000A"
    20Min_Hoog: " I --- 37:012161 37:252531 --:------ 22F3 003 000014"
    30Min_Hoog: " I --- 37:012161 37:252531 --:------ 22F3 003 00001E"
  _note: based upon RFT-N Auto with itho cve-s eco
"37:252531":
  class: FAN
  _note: Itho CVE-S ECO

```

**Comment by zxdavb** [comment]
I think I fixed this in release 0.50.0

**Comment by hermanbanken** [comment]
I also had this in v0.42.0. Will auto now to verify if fixed.
